cvat-core/src/quality-settings.ts

```diff
@@ -61,151 +88,171 @@ export default class QualitySettings {
         this.#panopticComparison = initialData.panoptic_comparison;
         this.#compareAttributes = initialData.compare_attributes;
         this.#emptyIsAnnotated = initialData.empty_is_annotated;
+        this.#jobFilter = initialData.job_filter || '';
+        this.#inherit = initialData.inherit;
         this.#descriptions = initialData.descriptions;
+
+        this.#updateTrigger = new FieldUpdateTrigger();
```

Comment:
I would deprecate FieldUpdateTrigger for further using.
It leads to uncertain state of the object when user just changes the fields, but not saves them on the server, or when settings were not saved successfully because of a reason. Also update behaviour can not be modified in plugin with this approach as updateTrigger is a private member. So, many potential problems. 

IMHO API should be:
settings.update({ ..fieldsToUpdate }) 

as we implemented for jobs.